src/proto/faabric.proto

```diff
@@ -242,3 +245,20 @@ message PointToPointMappings {
 
     repeated PointToPointMapping mappings = 3;
 }
+
+// ---------------------------------------------
+// FUNCTION MIGRATIONS
+// ---------------------------------------------
+
+message PendingMigrations {
+    int32 appId = 1;
+    int32 groupId = 2;
+
+    message PendingMigration {
+        Message msg = 1;
```

Comment:
AFAICT this is the message that's sent to worker hosts to tell it to migrate a function across hosts. If that's right, doesn't that host already have the messge, so we just need the message ID here?

Reply:
All functions in the app will read the same `PendingMigrations` message. They:
* Iterate over the `PendingMigration` repeated field (`migrations`).
* If the message corresponds to their message id, they themselves must trigger their own migration.
* To do so, they need to know where they are headed `dstHost`.

Maybe the `srcHost` field is not strictly needed, but it can be asserted/makes printing and logging easier.